experiments/zfs-localpv/functional/backup_and_restore/run_litmus_test.yml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: configmap-zfspv-backup-store
+  namespace: litmus
+data:
+  parameters.yml: |
+    
+---
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: zfspv-velero-backup-restore-
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        app: velero-backup-restore
+
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: openebs/ansible-runner:ci
+        imagePullPolicy: IfNotPresent
+
+        env:
+          - name: ANSIBLE_STDOUT_CALLBACK
+            value: default
+
+          - name: APP_NAMESPACE  ## Namespace where application is deployed
+            value: '' 
+
+          - name: APP_LABEL  ## Application label
+            value: ''
+
+          - name: APP_PVC  ## PVC name for application
+            value: ''
+
+          - name: ACTION  ## On basis of the value of this env, this playbook will perform tasks.
+            value: ''     ## Supported values: (backup, restore and schedule_backup)
+          
+          - name: VELERO_BACKUP_NAME  ## Velero backup will be created with name of this value.
+            value: ''
+
+          - name: VELERO_PLUGIN_NAME  ## Velero plugin name. For zfs-localpv plugin name is `openebs/velero-plugin:2.1.0`
+            value: ''                 ## you can use ci image also. 
+
+          - name: VELERO_SCHEDULE_BACKUP_NAME  ## If you want to create scheduled backup (based on ACTION env) give the name for it.
+            value: ''
+
+          - name: VELERO_VERSION   ## Velero version (for e.g. v1.4.0)
+            value: ''
+
+          - name: STORAGE_BUCKET  ## Supported values: minio
```

Comment:
is minio alone supported?

Reply:
As of this automated experiment only minio...will add other buckets also after verifying